assets/styles/layouts/_Footer.scss

```diff
@@ -138,19 +138,30 @@
 
 		li {
 
-			svg path {
+			a {
 
-				@include color(fill, font-color-medium);
-				transition: fill 0.5s;
-			}
+				svg {
+
+					@include square(1.2em);
+
+					path {
+
+						@include color(fill, font-color-medium);
+						transition: fill 0.5s;
+					}
+				}
 
-			a:hover {
+				&:active,
+				&:focus,
+				&:hover {
 
-				@include color(color, primary-color);
+					@include color(color, primary-color);
+					text-decoration: none;
 
-				svg path {
+					svg path {
```

Comment:
svg without path is enough. If there is no fill color set (fill="none" is also considered as set) then svg takes currentColor specified via color: property

Reply:
paht selector was removed